src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -557,7 +557,7 @@
           "Log cumulative card stats every so many remembered set or "      \
           "update refs scans")                                              \
                                                                             \
-  product(uintx, ShenandoahMinimumOldMarkTimeMs, 100, EXPERIMENTAL,         \
+  product(uintx, ShenandoahMinimumOldTimeMs, 100, EXPERIMENTAL,         \
          "Minimum amount of time in milliseconds to run old marking "       \
```

Comment:
May want to adjust the "description": Minimum time to run old marking or old cleanup (C&F could be considered cleanup - or just say C&F)...

Reply:
Yep, good catch.